src/core/entities/MRImageEntity.js

```diff
@@ -43,7 +43,8 @@ export class MRImageEntity extends MRMediaEntity {
      * @function
      * @description Callback function of MREntity - handles setting up this Image and associated 3D geometry style (from css) once it is connected to run as an entity component.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?